ui/src/app/workflows/components/workflows-row/workflows-row.tsx

```diff
@@ -61,10 +63,10 @@ export function WorkflowsRow(props: WorkflowsRowProps) {
                     </Link>
                     <div className='columns small-1'>{wf.metadata.namespace}</div>
                     <div className='columns small-1'>
-                        <Timestamp date={wf.status.startedAt} />
+                        <Timestamp date={wf.status.startedAt} displayISOFormat={props.displayISOFormatStart} />
```

Comment:
Hello, @Adrien-D, `small-1` class cannot fit the entire date. Could you make it wider if possible?

![image](https://github.com/user-attachments/assets/a5128f48-62f9-4c5f-9237-ca3e97b435ef)

Reply:
@djanjic depending on which table yes we could, so I'll update when possible. But for workflow list for example not really as we already have 10 columns with the first two being `small-2`

Reply:
I've change to `small-2` for cron and here is what it looks like for workflows 

<img width="1327" alt="Screenshot 2024-07-29 at 15 50 49" src="https://github.com/user-attachments/assets/adfe2bd3-320b-4ca9-9ffc-4bfa5bdd756e">